### ISSUE TITLE
How should multi-networks be handled?

### ISSUE THREAD

**Issue by noahrhodes** [WISPO-POP/PowerPlots.jl#33]
1. Dis-allow multi-networks?
 - require a user to pass a single network from the multi-network
 
2. Create a separate plot for each network automatically?

3. Use html to create a window for embedding plots, with interaction to click to each network

(ranked by increasing difficult)

Any other ideas?

**Comment by noahrhodes** [comment]
Use a slider to change a value that filters the data according to the multi-network id?

https://vega.github.io/vega-lite/docs/bind.html

**Comment by noahrhodes** [comment]
Here is a vegalite code example of using a filter connected to a slider.  Can use this to filter the network id of a multinetwork
```javascript
{
  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
  "description": "Drag the sliders to highlight points.",
  "data": {"url": "data/cars.json"},
  "transform": [
      {"calculate": "year(datum.Year)", "as": "Year"},
      {"filter": {"param": "CylYr"}}
    ],
  "layer": [{
    "params": [{
      "name": "CylYr",
      "value": [{"Cylinders": 4, "Year": 1977}],
      "select": {"type": "point", "fields": ["Cylinders", "Year"]},
      "bind": {
        "Cylinders": {"input": "range", "min": 3, "max": 8, "step": 1},
        "Year": {"input": "range", "min": 1969, "max": 1981, "step": 1}
      }
    }],
    "mark": "circle",
    "encoding": {
      "x": {"field": "Horsepower", "type": "quantitative"},
      "y": {"field": "Miles_per_Gallon", "type": "quantitative"},
      "color": {
        "condition": {"param": "CylYr", "field": "Origin", "type": "nominal"},
        "value": "grey"
      }
    }
  }, {
    "transform": [{"filter": {"param": "CylYr"}}],
    "mark": "circle",
    "encoding": {
      "x": {"field": "Horsepower", "type": "quantitative"},
      "y": {"field": "Miles_per_Gallon", "type": "quantitative"},
      "color": {"field": "Origin", "type": "nominal"},
      "size": {"value": 100}
    }
  }]
}
```

**Comment by noahrhodes** [comment]
closed by #86
